packages/scss/src/components/inputs/textfield/_textfield.framed.scss

```diff
@@ -30,6 +50,7 @@
 
 	.textfield-suffix {
 		bottom: _component("textfield.framed.bottom-padding");
+		position: absolute;
```

Comment:
It should not be necessary when you put the original definition back in textfield.scss

Reply:
✔️